components/tables.html

```diff
@@ -84,6 +84,23 @@
         <li>Make sure that any row that is sortable has the sort hover states</li>
         <li>It’s helpful to include an information icon with a tooltip that explains the sorting function on sortable rows</li>
       </ul>
+      <ul>
```

Comment:
Any reason for ending one list only to immediately start a new one? If this list was supposed to go under that last bullet, take out `</ul>` on line 86 and move the `</li>` from the end of line 85 to a new line after line 103.